site/slices/checklist-contact-us-slice/index.js

```diff
@@ -49,15 +49,18 @@ class ChecklistContactUs extends Component {
           </ul>
           <div className={cx(styles.imgContainer, this.state.isHovered ? 'isHovered' : '')} />
         </div>
-        <a
-          href={mailToURL}
-          className={styles.button}
-          onMouseEnter={this.onHover}
-          onMouseLeave={this.onBlur}
-          onClick={trackAnalytics('ContactUsForm - ButtonClicked')}
-        >
-          Talk to us
-        </a>
+        <div>
+          <a
+            href={mailToURL}
+            className={styles.mailToLink}
+            onMouseEnter={this.onHover}
+            onMouseLeave={this.onBlur}
+            onClick={trackAnalytics('ContactUsForm - ButtonClicked')}
+          >
+            Send an email
+          </a>
+          <h3 className={styles.talkToUs}>hello@red-badger.com</h3>
```

Comment:
Dont think this should be an h3, as these usually refer to a opening of an section.